### ISSUE TITLE
Add License

### ISSUE THREAD

**Issue by henryreed** [IppSec/ippsec.github.io#23]
Hello Ippsec and contributors,

I'd like to copy this template to use for cocktail recipes from a completely different YouTube channel. I'd like to know what I can and cannot do with the code and would prefer to have a license to work with. If possible, please add a license to this repository. Thanks!

**Comment by ippsec-htb** [comment]
Hey @henryreed sorry for the lateness. Can you recommend a license to use? You are free to use any of the code here anyway you want. I'd appreciate if you gave some credit, but its not required.

**Comment by henryreed** [comment]
Hi @ippsec-htb! No worries at all--thanks for your time.

If you want to completely abstain from legally requiring folk to credit you, there is the [Creative Commons CC0-1.0](https://creativecommons.org/publicdomain/zero/1.0/), as well as the [Zero-Clause BSD](https://opensource.org/licenses/0BSD). There is some minor discussion of licenses like that [here](https://en.wikipedia.org/wiki/Public-domain-equivalent_license), if you'd like some additional reading. The nice thing about the Creative Commons CC0-1.0 license is that it attempts to waive copyright in countries where this is legal, like the United States, and in countries where it's not, like Germany, it has a fallback to a ["public domain equivalent license"](https://en.wikipedia.org/wiki/Public_domain_equivalent_license).

I am planning to give credit, no matter the license :)

**Comment by IppSec** [comment]
I have added license.txt to the root of this repository, which is the CC0 license. I don't want to add it to the actual website due to the website not having any external links.  I've tried to keep it pretty minimal (minus the horrendously inefficient json db that I always intended to clean up but never did) 

Let me know if there's a better way to do it, I actually never thought about licenses before.
